etc/bash_completion.d/juju2

```diff
@@ -1,42 +1,55 @@
+#!/bin/bash
 # juju-core.bash_completion.sh: dynamic bash completion for juju cmdline,
 # from parsed (and cached) juju status output.
 #
 # Author: JuanJo Ciarlante <jjo@canonical.com>
 # Copyright 2013+, Canonical Ltd.
 # License: GPLv3
 #
+# For juju2, includes --model and --controller handling:
+#   juju list-models --controller <TAB>
+#   juju switch <TAB>
+#   juju status --model <TAB>
+#   juju ssh --model <TAB> [... will complete with proper model's units/etc ...]
+#
 
 # Print (return) all machines
-_juju_machines_from_file() {
-python -c '
+_juju_machines_from_status_file() {
+python3 -c '
```

Comment:
We want this script to work on backports to trusty etc, which may have python3 available, but not installed. Can we also have a $PYTHON variable that gets set the appropriate default for the system?